cirq-core/cirq/ops/raw_types.py

```diff
@@ -319,8 +319,8 @@ def __pow__(self, power):
 
         return NotImplemented
 
-    def __call__(self, *args, **kwargs):
-        return self.on(*args, **kwargs)
+    def __call__(self, *qubits: Qid, **kwargs):
+        return self.on(*qubits)
```

Comment:
You also need to pass the **kwargs to `self.on()` call, so that `cirq.CNOT(control=a, target=b)` works ?

Reply:
The `cirq.Gate` class defines the `on` method as:

```python
def on(self, *qubits: Qid) -> 'Operation':
    ....
```
whereas it's derived class, `CXPowGate`, overrides the `on` method as 

```
def on(self, *args: 'cirq.Qid', **kwargs: 'cirq.Qid') -> raw_types.Operation:
    ....
```

Technically, this does satisfy the liskov substitution principle because the derived class is implementing a more generic method (that accepts both *args and **kwargs) compared to the base class (that can accept only *args). 
I think your current solution, to not specify type of `**kwargs` in `__call__` and silently forwarding it to `self.on()`, should work. 

One other way could be to enforce that the `cirq.Gate`'s `__call__`  method will only accept `*qubits` and whenever a derived class, like CXPowGate, overrides the `on` method to be more generic, it should also override the `__call__` method? But that would be a huge breaking change for all existing implementations, like `CXPowGate`, and something we probably don't want to do at this point.

Reply:
yes sorry thought I had fixed this.  This is what happens when check/all doesn't run tests in parallel so I just push it out without running the tests (I can haz review for #5006 ?)